src/containers/Navbar/smallScreenNavbarData.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  HouseDoor,
+  Search,
+  Gift,
+  Heart,
+  InfoCircle,
+  Envelope,
+} from "react-bootstrap-icons";
+
+export const smallScreenNavbarData = [
+  {
+    cName: "navLi",
```

Comment:
what does c stand for here ?

Reply:
@halitbatur className, didnt want to use syntax of react so i wrote cName

Reply:
@ceyhungulbas I think u can use className here since its treated as a key

Reply:
@ceyhungulbas and since all of them have the same className why not just write it there right away instead of having it in all the objects ?